### PR TITLE
Get rid of quoting that breaks by default.

### PR DESCRIPTION
**Pull request by Sheeri**
It looks like someone tried to use '' to make a ' happen for the possessive tense of users. But that messes up the quoting:
issue-message: 'Message that will be displayed on users'' first issue'
pr-message: 'Message that will be displayed on users'' first pr'

It was a very bad impression to have a simple script designed to welcome folks be broken by default.

It should be:
issue-message: 'Message that will be displayed on the first issue for that user'
pr-message: 'Message that will be displayed on the first pr for that user'

(this gets rid of the spurious quotes, but also doesn't introduce any grammatical errors)

-------
In the workflow and properties files:

- [x] The workflow filename of CI workflows should be the name of the language or platform, in lower case.  Special characters should be removed or replaced with words as appropriate (for example, "dotnet" instead of ".NET").

  The workflow filename of publishing workflows should be the name of the language or platform, in lower case, followed by "-publish".
- [x] Includes a matching `ci/properties/*.properties.json` file.
- [x] Use sentence case for the names of workflows and steps, for example "Run tests".
- [x] The name of CI workflows should only be the name of the language or platform: for example "Go" (not "Go CI" or "Go Build")
- [x] Include comments in the workflow for any parts that are not obvious or could use clarification.
- [x] CI workflows should run on `push` to `branches: [ $default-branch ]` and `pull_request` to `branches: [ $default-branch ]`.
- [x] Packaging workflows should run on `release` with `types: [ created ]`.

Some general notes:

- [x] This workflow must only use actions that are produced by GitHub, [in the `actions` organization](https://github.com/actions), **or**
- [x] This workflow must only use actions that are produced by the language or ecosystem that the workflow supports.  These actions must be [published to the GitHub Marketplace](https://github.com/marketplace?type=actions).  Workflows using these actions must reference the action using the full 40 character hash of the action's commit instead of a tag.  Additionally, workflows must include the following comment at the top of the workflow file:
    ```
    # This workflow uses actions that are not certified by GitHub.
    # They are provided by a third-party and are governed by
    # separate terms of service, privacy policy, and support
    # documentation.
    ```
- [x] This workflow must not send data to any 3rd party service except for the purposes of installing dependencies.
- [x] This workflow must not use a paid service or product.

### CODE REVIEW

File: automation/greetings.yml [hunk 1]
```diff
@@ -9,5 +9,5 @@ jobs:
     - uses: actions/first-interaction@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        issue-message: 'Message that will be displayed on users'' first issue'
-        pr-message: 'Message that will be displayed on users'' first pr'
+        issue-message: 'Message that will be displayed on users'
+        pr-message: 'Message that will be displayed on users'
```
Comment (andymckay) [inline automation/greetings.yml:13]:
```suggestion
        issue-message: 'Message that will be displayed on users first issue'
        pr-message: 'Message that will be displayed on users first pull request'
```

**Review by andymckay — APPROVED** [review]
Thanks for spotting this @Sheeri, I agree. Can we add in my suggestion, without it the sentence just trails off and doesn't make sense.

File: automation/greetings.yml (hunk 1 above)
Comment (Sheeri) [reply automation/greetings.yml:13]: Hi @andymckay - Indeed, we don't want the sentence to trail off. However, now it's gramatically incorrect. How about:

issue-message: 'Message that will be displayed on the first issue for that user'
pr-message: 'Message that will be displayed on the first pr for that user'

Comment (andymckay) [reply automation/greetings.yml:13]: Sure that works for me 👍🏾

**Comment by andymckay** [comment]
Added suggestions and merged, so we can remove this breakage for new users. Hope that's ok @Sheeri.

**Comment by Sheeri** [comment]
absolutely, thanks!

**Pull request merged**